AnnService/inc/Core/SPANN/ExtraFullGraphSearcher.h

```diff
@@ -552,7 +550,7 @@ namespace SPTAG
                 LOG(Helper::LogLevel::LL_Info, "EnableDeltaEncoding: %s\n", p_opt.m_enableDeltaEncoding ? "true" : "false");
                 LOG(Helper::LogLevel::LL_Info, "EnableDataCompression: %s, ZstdCompressLevel: %d\n", p_opt.m_enableDataCompression ? "true" : "false", p_opt.m_zstdCompressLevel);
                 std::vector<size_t> postingListBytes(headVectorIDS.size());
-                std::unique_ptr<Compressor> m_pCompressor = std::make_unique<Compressor>(p_opt.m_zstdCompressLevel);
+                auto m_pCompressor = std::make_unique<Compressor>(p_opt.m_zstdCompressLevel);
```

Comment:
Why do you use auto to replace std::unique_ptr?
I think std::unique_ptr is more clear.